src/client.ts

```diff
@@ -60,6 +60,9 @@ export interface Config {
 
 	/** Optional config for Amplitude client. */
 	amplitude?: Exclude<AmplitudeOverride, amplitude.Config>;
+
+	/** Optional device_id for Amplitude client. */
```

Comment:
Shall we make it possible to pass `userId` as well? 

It might help to initialize the analytics-client with a `userId` when the user is already logged in.

Reply:
Sounds like a good idea!

Reply:
Actually we cant, `user_id` is not a passable parameter to the `.init` method. The passable parameters are listed [here](https://amplitude.github.io/Amplitude-JavaScript/Options/#options)

Reply:
Right. Seems like we need to use `identify` for user identification. Thank for checking!